packages/cli/src/routeGeneration/templates/express.hbs

```diff
@@ -72,7 +72,14 @@ export function RegisterRoutes(app: express.Express) {
             }
 
             {{#if ../../iocModule}}
-            const controller: any = iocContainer.get<{{../name}}>({{../name}});
+            function isFunction(obj: any): obj is Function { return typeof obj === 'function'; }
+            interface Container<T> { get<T>(type: any): any; }
```

Comment:
Again, the `Container` interface here is to satisfy TypeScript. Without it, `container<Type>.get` has the error 'Untyped function calls may not accept type arguments.'